build/CoreFoundation/dll/CoreFoundation.def

```diff
@@ -8,6 +8,9 @@ LIBRARY CoreFoundation
         __objc_class_name__NSCFNumber CONSTANT
         _OBJC_CLASS__NSCFBoolean DATA
         __objc_class_name__NSCFBoolean CONSTANT
+	; Exposed for other CoreFoundation bridging implementors
+        _OBJC_CLASS__NSCFType DATA
```

Comment:
This doesn't look like it's being used in CoreGraphics (unless I missed something). Is this in preparation for something else? If so, shouldn't it be checked in alongside that?

Reply:
(rebuilding to verify why I did this)

Reply:
This is required in CoreText, actually, because CT uses `CD_OBJC_FUNCDISPATCHV`, which uses `CF_IS_OBJC`, which uses `NSCFType`